### DISCUSSION TITLE
Blank / Empty Output?

### DISCUSSION THREAD

**Discussion by ThioJoe** [oobabooga/text-generation-webui#1421] [Q&A]
I'm trying to get a 65B model working. I have a 4090 and a 3090. I have gotten many other models working from 7B to 30B, all outputting fine. But for the 65B one's I've tried, it basically outputs nothing. I think once or twice it generated a period and dash. 

Specifically the ones I've tried are:
• Neko-Institute-of-Science_LLaMA-65B-4bit-128g
• TheBloke_alpaca-lora-65B-GPTQ-4bit

On the second one for example, the console shows it apparently was trying to generate an output:
`Output generated in 37.08 seconds (5.37 tokens/s, 199 tokens, context 49, seed 1598353902)`

My run command looks like this: 
`call python server.py --auto-devices --gpu-memory 23 23 --chat --model TheBloke_alpaca-lora-65B-GPTQ-4bit --wbits 4 --groupsize 128`
But I've also tried without specifying the GPU memory. It starts in both cases, but again no output.

I've also tried using some of the 30B models and forcing each GPU to have 12 GB VRAM to see if splitting the model across them was the problem, but that still works, just not the 65B.

**Comment by jpoles1** [comment]
I'm experiencing the same issue with TheBloke_alpaca-lora-65B-GPTQ-4bit
